docs/src/data/mockServices.tsx

```diff
@@ -0,0 +1,89 @@
+import { Amplify} from 'aws-amplify';
+
+const awsExports = {
+  aws_user_pools_id: 'xx-xxxx-x_xxxxx', // (optional) -  Amazon Cognito User Pool ID
+  aws_user_pools_web_client_id: 'xxxxxxxxxxxxxx', // (optional) - Amazon Cognito App Client ID (App client secret needs to be disabled)
+};
+Amplify.configure(awsExports)
+
+let signUpPassword = "";
+const verifiedUsers = new Map();
+const fixed_username = "amplify-ui";
+const fixed_password = "connected-components";
+
+export const mockServices = {
+  async getAmplifyConfig() {
+    return {};
+  },
+  async handleSignUp(formData) {
+    if(verifiedUsers.has(formData.username)) {
+      return Promise.reject(new Error("User already exists!"))
```

Comment:
Is this really necessary? I don't think we should build in too much service level logic that we then have to maintain. I think our demos should show off the UI screens, not perfectly match the service behaviors.

Reply:
Talked with Scott and fixed this!

Reply:
Curious about this too, what changes are we making?

Reply:
Basically, if a user tries to sign up with an existing username, it won't throw any error now.